src/Tests.Microsoft.Security.Utilities.Core/IdentifiableSecretsTests.cs

```diff
@@ -1028,8 +1028,8 @@ public void IdentifiableSecrets_TryValidateBase64Key_ChecksAlphabet()
             bool result;
             ulong seed = 42;
             string signature = "TEST";
-            string secretBase64Std = "83Tv3p0dtmc/cw7eEVxcC7lh6ZM+MgPG3TESTLBLKt4=";
-            string secretBase64Url = "83Tv3p0dtmc_cw7eEVxcC7lh6ZM-MgPG3TESTLBLKt4=";
```

Comment:
>MgPG3TESTLBLKt4 [](http://example.com/codeflow?start=66&length=15)

You can formally define test keys in your specification but that doesn't mean anyone else will know about them. :)